ua-snap/swti
# Update normals

Closes #66

Adds a script that will rebuild the normals, as well as including the updated normals file.

To test this:
 - Check the timestamp on the `data/normals.csv` file.  Then run the script in the `data` directory (from within that directory) with `pipenv run python rebuild_normals.py`.  It should complete and you'll see an updated timestamp, but the data values shouldn't be any different from what's in the repo now.
 - Run the app locally _not_ in debug mode, i.e. don't have `FLASK_DEBUG` set.  Load the app, then save the generated image (rename it to be clear on which is which).  Do the same on the live/production version.  Compare the two in Preview -- they will be similar but slightly different.
